velox/exec/Operator.cpp

```diff
@@ -115,10 +115,32 @@ void Operator::maybeSetTracer() {
     return;
   }
 
-  if (queryTraceConfig->queryNodes.count(planNodeId()) == 0) {
+  const auto nodeId = planNodeId();
+  if (queryTraceConfig->queryNodes.count(nodeId) == 0) {
```

Comment:
You can just use operator id here since it is per driver ctx? And no need for lock as it is single threaded. Thanks!

Reply:
Got it, fixed.